.jenkins/Jenkinsfile

```diff
@@ -160,15 +160,10 @@ def DCAPBuildTest(String label, String build_type) {
     }
 }
 
-parallel "ACC1604 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1604', '16.04') },
-         "ACC1804 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1804', '18.04') },
-         "ACC1604 SGX1-FLC clang-8 Debug" :            { ACCTest('ACC-1604', '16.04', 'clang-8', 'Debug') },
-         "ACC1604 SGX1-FLC clang-8 Release" :          { ACCTest('ACC-1604', '16.04', 'clang-8', 'Release') },
-         "ACC1604 SGX1-FLC clang-8 RelWithDebInfo" :   { ACCTest('ACC-1604', '16.04', 'clang-8', 'RelWithDebInfo') },
-         "ACC1804 SGX1-FLC clang-8 Debug" :            { ACCTest('ACC-1804', '18.04', 'clang-8', 'Debug') },
-         "ACC1804 SGX1-FLC clang-8 Release" :          { ACCTest('ACC-1804', '18.04', 'clang-8', 'Release') },
-         "ACC1804 SGX1-FLC clang-8 RelWithDebInfo" :   { ACCTest('ACC-1804', '18.04', 'clang-8', 'RelWithDebInfo') },
-         "ACC1604 OpenEnclave Release Test" :          { ACCTestOeRelease('ACC-1604','16.04') },
+parallel "ACC1804 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1804', '18.04') },
+         "ACC1804 SGX1-FLC clang-10 Debug" :            { ACCTest('ACC-1804', '18.04', 'clang-10', 'Debug') },
+         "ACC1804 SGX1-FLC clang-10 Release" :          { ACCTest('ACC-1804', '18.04', 'clang-10', 'Release') },
```

Comment:
`Release` build type is not recommended, and `RelWithDebInfo` or `Debug` should be used